static/js/components/search/FilterVisibilityToggle.js

```diff
@@ -15,13 +17,29 @@ export default class FilterVisibilityToggle extends React.Component {
     return checkFilterVisibility(filterName) ? "" : "closed";
   };
 
-  openStateIcon: Function = (): React$Element<*> => (
-    <Icon
+  isInResults: Function = (id: string): boolean => {
+    if (this.getResults()) {
+      const elmId = (id === "birth_location") ? "profile.birth_country3" : id;
+      const docCount = _.get(this.getResults(), ['aggregations', elmId, 'doc_count']) || 0;
+
+      if (docCount > 0) {
+        return true;
+      }
+    }
+    return false;
+  }
+
+  openStateIcon: Function = (children: React$Element<*>): React$Element<*>|null => {
+    if (this.getResults() === null || !this.isInResults(children.props.id)) {
```

Comment:
The code already does the `this.getResults()` check in `isInResults`, can you remove that part of the expression?